client/src/components/ui/Typography/Typography.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import { theme } from "../../../theme/theme";
+
+interface TypographyProps {
+	fontSize?: number;
+	fontWeight?: number;
+}
+
+export const Typography = styled.p<TypographyProps>`
```

Comment:
I think you need to change this part of code for sth like this(look below), because now we have only 2 options for font-size and font-weight and at the theme there is much more possibilities 

`export const Typography = styled.p<TypographyProps>`
  font-size: ${({ fontSize }) => (fontSize ? fontSize : theme.size.desktop.sm)};
  font-weight: ${({ fontWeight }) =>
    fontWeight ? fontWeight : theme.weight.regular};

  @media (max-width: 576px) {
    font-size: ${({ fontSize }) =>
      fontSize ? fontSize : theme.size.mobile.sm};
  }
`;`